pom.xml

```diff
@@ -6,37 +6,26 @@
 
     <groupId>io.qameta.allure.examples</groupId>
     <artifactId>allure-junit5-example</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.1</version>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
-        <junit.api.version>5.3.1</junit.api.version>
-        <junit.jupiter.version>5.3.1</junit.jupiter.version>
-        <junit.platform.version>1.3.1</junit.platform.version>
-        <aspectj.version>1.9.2</aspectj.version>
         <allure.version>2.8.1</allure.version>
     </properties>
 
     <dependencies>
-        <!-- Test API(s) -->
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <version>${junit.api.version}</version>
```

Comment:
it is better to keep `junit-jupiter-api`. Otherwise you need to make sure `allure-junit5` depends on the same version of it.

All we need is extend [official example](https://github.com/junit-team/junit5-samples/blob/r5.3.1/junit5-jupiter-starter-maven/pom.xml) and add minimal Allure configuration:

* allure-junit5 dependency 
* aspectj configuration to add `@Step` and `@Attachment` annotations support
* allure-maven plugin to generate the report

Reply:
> It is better to keep junit-jupiter-api. Otherwise you need to make sure allure-junit5 depends on the same version of it.
> 
> All we need is extend official example 

I am not quite getting it, junit-jupiter-engine already depends on junit-jupiter-api and downloading it automatically...

Reply:
In general, you are right. 

A little note:

`junit-jupiter-engine` is only required in runtime, but as long as Maven has no support of `testRuntime` scope we are using `test` instead. So in perfect would it should be 

```
testCompile('org.junit.jupiter:junit-jupiter-api:5.3.1')
testRuntime('org.junit.jupiter:junit-jupiter-engine:5.3.1')
```

As I said before, we'd like to stick with official example, they have both dependencies added.
Also the problem I mentioned before is that `allure-junit5` depends on `junit-jupiter-api` as well. So in order to prevent dependency hell (`allure-junit5` can use older version of junit-api at some point) you need to specify `junit-jupiter-api` dependency directly in your pom.xml